content/docs/1.4.5/references/settings.md

```diff
@@ -647,6 +648,18 @@ Notice that trying to trim a removed files from a valid snapshot will do nothing
 
 See [Trim Filesystem](../../volumes-and-nodes/trim-filesystem) for details.
 
+#### Disable Snapshot Purge
+
+> Default: `false`
+
+When set to true, temporarily prevent all attempts to purge volume snapshots.
+
+Longhorn typically purges snapshots during replica rebuilding and user-initiated snapshot deletion. During this process,
+Longhorn coalesces unnecessary snapshots into their newer counterparts, freeing space consumed by historical data. It
+is always desirable to allow snapshot purging during normal operation. However, purging does temporarily consume
+additional disk space. If an underlying disk is too full for a snapshot purge operation to complete successfully, it may
+be beneficial to temporarily disable purging while data is moved to other disks.
```

Comment:
```suggestion
Longhorn typically purges snapshots during replica rebuilding and user-initiated snapshot deletion. While purging, Longhorn coalesces unnecessary snapshots into their newer counterparts, freeing space consumed by historical data.

Allowing snapshot purging during normal operations is ideal, but this process temporarily consumes additional disk space. If insufficient disk space prevents the process from continuing, consider temporarily disabling purging while data is moved to other disks.
```

Reply:
I suggest having two paragraphs so users can scan the text more easily. The first one explains concepts while the second one provides recommendations. I also reworded some parts to reduce the word count.

Reply:
No concerns. Fixed by amending the commit.